packages/webhooks/schema/models/WebhookSubscription.js

```diff
@@ -0,0 +1,123 @@
+const { Json } = require('@condo/keystone/fields')
+const { uuided, versioned, tracked, softDeleted, dvAndSender, historical } = require('@condo/keystone/plugins')
+const { GQLListSchema } = require('@condo/keystone/schema')
+const access = require('@condo/webhooks/schema/access/WebhookSubscription')
+const { WebHookModelValidator, getModelValidator, setModelValidator } = require('@condo/webhooks/model-validator')
+
+function getWebhookSubscriptionModel (schemaPath) {
+    if (!getModelValidator()) {
+        setModelValidator(new WebHookModelValidator(schemaPath))
+    }
+
+    const validator = getModelValidator()
+
+    const validateFields = ({ resolvedData, existingItem, addFieldValidationError }) => {
+        const newItem = { ...existingItem, ...resolvedData }
+
+        if (!validator) {
+            return addFieldValidationError(`Invalid fields for model "${newItem.model}" was provided. Details: ["Validator for this model is not specified!"]`)
+        }
+
+        const { isValid, errors } = validator.validateFields(newItem.model, newItem['fields'])
+        if (!isValid) {
+            const errorMessage = errors.map(error => {
+                if (typeof error === 'string') {
+                    return `"${error}"`
+                } else {
+                    return JSON.stringify(error)
+                }
+            }).join(', ')
+            return addFieldValidationError(`Invalid fields for model "${newItem.model}" was provided. Details: [${errorMessage}]`)
+        }
+    }
+
+    const validateFilters = ({ resolvedData, existingItem, addFieldValidationError }) => {
+        const newItem = { ...existingItem, ...resolvedData }
+
+        if (!validator) {
+            return addFieldValidationError(`Invalid filters for model "${newItem.model}" was provided. Details: ["Validator for this model is not specified!"]`)
+        }
+
+        const { isValid, errors } = validator.validateFilters(newItem.model, newItem['filters'])
+        if (!isValid) {
+            return addFieldValidationError(`Invalid filters for model "${newItem.model}" was provided. Details: ${JSON.stringify(errors)}`)
+        }
+    }
+
+
+    return new GQLListSchema('WebhookSubscription', {
+        schemaDoc: 'Determines which models the WebHook will be subscribed to. When model changes subscription task will be triggered to resolve changed data and send a webhook',
+        fields: {
+            webhook: {
+                schemaDoc: 'Link to a webhook containing information about integration',
+                type: Relationship,
+                ref: 'Webhook',
+                isRequired: true,
+                knexOptions: { isNotNullable: true },
+                kmigratorOptions: { null: false, on_delete: 'models.CASCADE' },
+            },
+            url: {
+                schemaDoc: 'Webhook target URL to which requests will be sent. Overrides url from webhook relation. ' +
```

Comment:
Why no just to create another webhook?

Reply:
Webhook will contain additional info about integration, such as secrets for encryption and etc